includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -165,9 +165,23 @@ private function get_argv_defaults( Check_Result $result ): array {
 			'--report-width=9999',
 		);
 
+		$ignore_patterns = array();
+
 		$directories_to_ignore = Plugin_Request_Utility::get_directories_to_ignore();
+		$files_to_ignore       = Plugin_Request_Utility::get_files_to_ignore();
+
+		// Ignore directories.
 		if ( ! empty( $directories_to_ignore ) ) {
-			$defaults[] = '--ignore=*/' . implode( '/*,*/', $directories_to_ignore ) . '/*';
+			$ignore_patterns[] = '*/' . implode( '/*,*/', $directories_to_ignore ) . '/*';
+		}
+
+		// Ignore files.
+		if ( ! empty( $files_to_ignore ) ) {
+			$ignore_patterns[] = implode( ',', $files_to_ignore );
```

Comment:
Do we need to use a pattern here as well to prevent the same `foobar.php` / `bar.php` partial match scenario?